doc/man3/DES_random_key.pod

```diff
@@ -104,9 +104,11 @@ consists of 8 bytes with odd parity.  The least significant bit in
 each byte is the parity bit.  The key schedule is an expanded form of
 the key; it is used to speed the encryption process.
 
-DES_random_key() generates a random key.  The PRNG must be seeded
-prior to using this function (see L<RAND_bytes(3)>).  If the PRNG
-could not generate a secure key, 0 is returned.
+DES_random_key() generates a random key.  The random generator must be
+seeded, otherwise the operation will fail.
+(The OpenSSL CSPRNG is seeded automatically on first use, but seeding can
+fail due to external circumstances, see L<RAND(7)>.)
+If the function could not generate a secure key, 0 is returned.
```

Comment:
I'd omit this line, we know there are situations where (on some operating systems soon after boot) the seeding isn't secure.

Reply:
I'll just write

```
If the function fails, 0 is returned.
```

Reply:
See 90139bf250d24f012b3f69c35c80727470e316a3.